libafl_qemu/build_linux.rs

```diff
@@ -12,6 +12,9 @@ pub fn build() {
             "usermode".to_string()
         })
     };
+
+    let asan = cfg!(feature = "asan");
```

Comment:
This feature is not in the Cargo.toml, it seems to always disable libqasan. You should add it, maybe rename it to build_libqasan, to the Cargo.toml and to the default features set

Reply:
Sorry, I'm not very familiar with the Rust build system. Could you take another look and check if this is what you had in mind?

Reply:
Yes, lgtm

Reply:
Is there a particular reason why libqasan cannot be compiled for hexagon? Maybe add a small comment in the build.rs to explain it otherwise we may forget about it in the future